### PR TITLE
This Pointer and Inheritance

### PR DESCRIPTION
**Pull request by PiyushNegi1703**
Python has an alternative of `this` which is `self` which I will be using for my project as an alternative of `this`.

### CODE REVIEW

**Comment by PiyushNegi1703** [comment]
The videos for the explanation of my code.
Part 1 - https://www.loom.com/share/0af16d58854f4e4d8c570ec000ce24ad?sid=2cbd2990-b8ad-4df6-967a-e5473d6b3642
Part 2 - https://www.loom.com/share/e38a4613430f45b1a156a954c4a13cba?sid=dcc81e30-0916-45fa-842e-f7c59572ea97
Part 3 - https://www.loom.com/share/74ff890376c34afa9cef00b807b68d91?sid=37a23c2e-672e-4b05-a2a8-d39f4be6f31e

File: server/controllers/userController.py
Comment (th3c0d3br34ker) [inline server/controllers/userController.py:1]: consider splitting this file into `router`, `controller` and `services`.

**Review by th3c0d3br34ker — COMMENTED** [review]
thanks @PiyushNegi1703 for the submission.

the loom covers the concepts of `this` pointer and `Inheritance`. 

however, a working demo is missing in the self-viva. please add that and let me know.

thanks.

File: server/controllers/userController.py
Comment (PiyushNegi1703) [reply server/controllers/userController.py:1]: The modular structure is `router`, `controller`, `models`, and `schemas`

**Comment by PiyushNegi1703** [comment]
Working demo for the milestone - https://www.loom.com/share/1764aa4ee4ab4ed39ea843feb0468528?sid=27cc8cad-3b76-47cb-84e0-549787552449

**Comment by th3c0d3br34ker** [comment]
thanks, @PiyushNegi1703 for reverting. please go a head and merge this PR

**Review by th3c0d3br34ker — APPROVED** [review]

**Pull request merged**